src/Countries/Malaysia.php

```diff
@@ -0,0 +1,535 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use DateTime;
+use DateTimeZone;
+use Exception;
+use IntlDateFormatter;
+use Spatie\Holidays\Exceptions\InvalidYear;
+
+class Malaysia extends Country
+{
+    protected string $timezone = 'Asia/Kuala_Lumpur';
+
+    protected function __construct(
+        protected ?string $region = null,
+    ) {
+    }
+
+    public function countryCode(): string
+    {
+        return 'my';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge(
+            $this->fixHolidays($year),
+            $this->variableHolidays($year)
+        );
+    }
+
+    /**
+     * @param int $year
+     * @return array<string, CarbonImmutable>
+     */
+    protected function fixHolidays(int $year): array
+    {
+        return [
+            'Hari Pekerja' => CarbonImmutable::createFromDate($year, 5, 1, $this->timezone),
```

Comment:
Could you use the format like other Country classes do? The timezone part can be removed.